test/labeling/apply/test_lf_applier.py

```diff
@@ -93,9 +103,15 @@ def fp_memoized(x: DataPoint) -> int:
     def test_lf_applier_no_labels(self) -> None:
         data_points = [SimpleNamespace(num=num) for num in DATA]
         applier = LFApplier([h])
-        L = applier.apply(data_points)
+        L = applier.apply(data_points, progress_bar=False)
         np.testing.assert_equal(L, -1)
 
+    def test_lf_applier_numpy(self) -> None:
+        X = np.vstack((DATA, DATA)).T
```

Comment:
is this to explicitly create 2D array to differentiate from doctest?

Reply:
why not `X = DATA` and remove the indexing in the LF

Reply:
because the point is to test that it works with ndarrays